rocAL/rocAL_pybind/CMakeLists.txt

```diff
@@ -19,7 +19,10 @@
 # THE SOFTWARE.
 
 cmake_minimum_required(VERSION 3.0)
-project(rocal_pybind)
+project(rocAL_pybind)
```

Comment:
I think it is better to rename the directory to "python" or rocal_python". We need to change all directories to use all smaller case letters. Only for the main github repo it can have "rocAL"

Reply:
@rrawther can we do the name change in a separate PR?

Reply:
I think it is better to keep all small letters for project name (rocal_pybind)